CBLClient/Apps/CBLTestServer-Android/app/src/main/java/com/couchbase/CouchbaseLiteServ/AndroidContext.java

```diff
@@ -0,0 +1,59 @@
+package com.couchbase.CouchbaseLiteServ;
+
+import com.couchbase.javacommon.Context;
+import com.couchbase.javacommon.log.Log;
```

Comment:
My bad.  I thought you were subclassing Android context.  Can you please rename this to something other than just `Context`?  ... like LiteServContext, or something?

Reply:
renamed the Context and Logger with test server specific wording